spellbook/models/looksrare/ethereum/looksrare_ethereum_events.sql

```diff
@@ -154,4 +156,9 @@ LEFT JOIN  {{ ref('nft_ethereum_aggregators') }} agg ON agg.contract_address = t
 LEFT JOIN {{ source('prices', 'usd') }} p ON p.minute = date_trunc('minute', looks_rare.block_time)
     AND p.contract_address = currency_contract
     AND p.blockchain ='ethereum'
-LEFT JOIN {{ ref('tokens_ethereum_erc20') }} erc20 ON erc20.contract_address = currency_contract
\ No newline at end of file
+LEFT JOIN {{ ref('tokens_ethereum_erc20') }} erc20 ON erc20.contract_address = currency_contract
+WHERE tx.block_time > '2022-01-01'
+{% if is_incremental() %}
+-- this filter will only be applied on an incremental run
+AND looks_rare.block_time > now() - interval 2 days
```

Comment:
the if is incremental is helpful, thank you 👍 

one other point, it may be more efficient to move this if block into the looks_rare CTE itself (after line 38 and after line 64 on both tables) that way the CTE doesn't do full scan, then outer query filters, we can get ahead of the filtering